sky/skylet/providers/aws/config.py

```diff
@@ -36,6 +36,11 @@
 DEFAULT_SKYPILOT_INSTANCE_PROFILE = SKYPILOT + "-v1"
```

Comment:
Note to myself, these changes should be sync'ed to #1702

Reply:
Note to me too: These changes should be synced to #2485 as well.